README.md

```diff
@@ -25,410 +25,20 @@ to be completed
-    <li>update: renamed GPS to Location, since location can be received via network as well as via GPS</li>
-    <li>update: code in LocationState</li>
-    <li>added: location provider in Sattelite and Compass screen</li>
-  </ul>
-</p>
-<h4>
-	<a name="#0.8.5"></a>
-	0.8.5 (12. 4. 2014)
-	<a href="#0.8.5" class="section_anchor"></a>
-</h4>
-<p>
-  <ul>
-    <li>added: create error log when application crashes</li>
-    <li>update: application description</li>
-  </ul>
-</p>
-<h4>
-	<a name="#0.8.4"></a>
-	0.8.4 (10. 4. 2014)
-	<a href="#0.8.4" class="section_anchor"></a>
-</h4>
-<p>
-  <ul>
-    <li>fixed: application crashing when user switches back from another application</li>
-    <li>fixed: too large wherigo icons on maps</li>
-    <li>added: german translation of new features</li>
-  </ul>
-</p>
-<h4>
-	<a name="#0.8.3"></a>
-	0.8.3 (6. 4. 2014)
-	<a href="#0.8.3" class="section_anchor"></a>
-</h4>
-<p>
-  <ul>
-    <li>fixed: application was crashing when user switched back from another application</li>
-    <li>fixed: compass displays azimuth value</li>
-    <li>added: save game button</li>
-    <li>added: option to save game automatically</li>
-    <li>added: compass and maps can handle other visible things than zones</li>
-  </ul>
-</p>
-<h4>
-	<a name="#0.8.2"></a>
-	0.8.2 (29. 3. 2014)
-	<a href="#0.8.2" class="section_anchor"></a>
-</h4>
-<p>
-  <ul>
-    <li>choose WhereYouGo folder</li>
-    <li>vector maps can handle moving zones</li>
-    <li>compass can handle moving zones</li>
-    <li>compass displays target name</li>
-    <li>window titles in local language</li>
-    <li>icons modernization</li>
-  </ul>
-</p>
-<h4>
-	<a name="#0.8.1"></a>
-	0.8.1 (23. 3. 2014)
-	<a href="#0.8.1" class="section_anchor"></a>
-</h4>
-<p>
-  <ul>
-    <li>remember render schema in vector maps</li>
-    <li>fixed problem in openwig with displaying items after save/load</li>
-    <li>added "map" button to cartridge menu</li>
-    <li>maps display all visible zones in current cartridge</li>
-    <li>set GPS always on as default option</li>
-  </ul>
-</p>
-<h4>
-	<a name="#0.8.0"></a>
-	0.8.0 (13. 3. 2014)
-	<a href="#0.8.0" class="section_anchor"></a>
-</h4>
-<p>
-	<ul>
-		<li>added vector OSM support using <a href="https://code.google.com/p/mapsforge/">MapsForge</a></li>
-		<li>added point labels in vector maps</li>
-		<li>added Zone border in both Locus and vector maps</li>
-		<li>added "Cancel" button when starting wherigo</li>
-		<li>fixed starting a new game</li>
-		<li>fixed encoding in "About application"</li>
-		<li>option to display WhereYouGo icon in the statusbar while the application is running</li>
-		<li>changed cartridge loading message</li>
-		<li>logging messages are appended to existing log file</li>
-	</ul>
-</p>
+An overview of released beta and production versions and their changelog can be found on the [release page of this repository](https://github.com/cgeo/WhereYouGo/releases).
```

Comment:
I would also refer to https://github.com/biylda/WhereYouGo/releases for older releases.

Reply:
Should be obsolete now, that we have the most recent old version in our releases.